src/jsc/jsc_value.cpp

```diff
@@ -99,6 +99,20 @@ JSValueRef jsc::Value::from_object_id(JSContextRef ctx, const ObjectId& value)
     return jsc::Function::construct(ctx, object_id_constructor, args.size(), args.data());
 }
 
+template<>
+JSValueRef jsc::Value::from_uuid(JSContextRef ctx, const UUID& value)
+{
+    static jsc::String s_realm = "Realm";
+    static jsc::String s_uuid = "_UUID";
+
+    JSObjectRef global_object = JSContextGetGlobalObject(ctx);
+    JSObjectRef realm_constructor = jsc::Object::validated_get_constructor(ctx, global_object, s_realm);
+    JSObjectRef uuid_constructor = jsc::Object::validated_get_constructor(ctx, realm_constructor, s_uuid);
```

Comment:
This won't work when the app is in Chrome debugging mode, since the `_UUID` won't be available on the `Realm` constructor. This is a problem for the other BSON types as well.

Reply:
Yeah, this implementation is suffering the same limitations as `ObjectId` currently sees. I've planned to look at it tomorrow with @takameyer.